v2/thirdparty/serverless/with-aws-lambda/authorizer.mdx

```diff
@@ -20,7 +23,124 @@ Follow [step 1](./frontend) and [step 2](./backend-config).
 ## 2) Create a new file `index.js`
```

Comment:
Change title "Add code to the lambda function handler"

Reply:
Done.